### PR TITLE
35 update dependencies

### LINKED ISSUE #35: Update dependencies
For example update `node-uuid` to `uuid` to get rid of warning on `npm install`

### PR DESCRIPTION
**Pull request by Retro64**
close #35

### CODE REVIEW

**Comment by Retro64** [comment]
:+1:

File: integration-test/oauth-tooling/tokenCache.spec.ts [hunk 1]
```diff
@@ -345,9 +345,9 @@ describe('tokenCache', () => {
 
     let promise = tokenService.refreshAllTokens()
       .then((res) => {
-        return res.map(tokeninfo => tokeninfo['access_token']);
+        const tokens = res.map(tokeninfo => tokeninfo['access_token']);
+        return tokens.sort();
```
Comment (bzums) [inline integration-test/oauth-tooling/tokenCache.spec.ts]: Is there a special reason to sort here?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lib-oauth-tooling",
-  "version": "0.26.3",
+  "version": "0.26.4",
```
Comment (bzums) [inline package.json]: Lets rather go to `0.27.0`

File: integration-test/oauth-tooling/tokenCache.spec.ts (hunk 1 above)
Comment (Retro64) [inline integration-test/oauth-tooling/tokenCache.spec.ts]: The test fails randomly as the order of the resulting array created from an unsorted, async created object is not guaranteed. Sort enforces the order that can be matched to the expected result.

**Comment by bzums** [comment]
👍

**Comment by Retro64** [comment]
:+1:

**Pull request merged**